apps/back/src/Entity/User.php

```diff
@@ -122,4 +141,34 @@ public function setIsVerified(bool $isVerified): static
 
         return $this;
     }
-}
\ No newline at end of file
+
+    public function getUsername(): ?string
+    {
+        return $this->username;
+    }
+
+    public function setUsername(?string $username): void
+    {
+        $this->username = $username;
+    }
```

Comment:
Ca semble pas utilisé.

Reply:
ça semble utilisé par le formulaire de d'inscription :
"Could not determine access type for property "username" in class "App\Entity\User"."
Au submit de ce dernier sans le "setUserName".